spline-shadow/client-ui/src/app/services/lineage/lineage-graph.service.ts

```diff
@@ -52,10 +55,12 @@ export class LineageGraphService {
     let that = this
     return that.executionPlanControllerService.lineageUsingGETResponse(executionPlanId).pipe(
```

Comment:
This seems like it can be simplified.

Reply:
not really, this is the way to write call to services in angular using rxjs lib.